validators/RequestSchema.js

```diff
@@ -145,6 +145,35 @@ const DeleteMediaFileRequestSchema = Joi.object().keys({
   sha: Joi.string().required(),
 })
 
+const UpdateNavigationRequestSchema = Joi.object().keys({
+  content: Joi.object()
+    .keys({
+      logo: Joi.string().allow(""),
+      links: Joi.array()
+        .items(
+          Joi.object()
+            .keys({
+              title: Joi.string().required(),
+              url: Joi.string(),
+              collection: Joi.string(),
```

Comment:
double checkin on this - they are optional and when not specified, will be missing from the keys of the object?

Reply:
Yep - we specify that at least one of them exist via the oxor requirements afterwards, but we normally only expect only a single one of `url`/`collection`/`resource_room`/`url` + `sublinks`